### PR TITLE
Support bbox filters in advanced search

### PR DESCRIPTION
**Pull request by tsauerwein**
The point geometry of documents is now indexed in ElasticSearch and bbox searches can be made on list views, e.g. `/waypoints?bbox=699398,5785365,799498,5885465`.

### CODE REVIEW

**Comment by asaunier** [comment]
👍

File: c2corg_api/search/search_filters.py [hunk 1]
```diff
@@ -184,6 +189,33 @@ def create_date_range_filter(field, query_term):
...
+    xmax, ymax = transform(bbox3857[2], bbox3857[3])
+
+    return GeoBoundingBox(
+        geom={'left': xmin, 'bottom': ymin, 'right': xmax, 'top': ymax},
+        type='indexed'
+    )
+
+
+@lru_cache(maxsize=1)
+def get_transform():
```
Comment (gberaudo) [inline c2corg_api/search/search_filters.py]: Have you considered using a file property?

File: requirements.txt [hunk 2]
```diff
@@ -14,7 +14,7 @@ Shapely==1.5.13
 pyproj==1.9.5.1
 pyramid-jwtauth==0.1.3
 bcrypt==2.0.0
-elasticsearch==2.1.0
+elasticsearch==2.2.0
```
Comment (gberaudo) [inline requirements.txt]: Shouldn't it be `2.3.2`?

**Comment by gberaudo** [comment]
I added some minor comments, otherwise looks good to me.

File: requirements.txt (hunk 2 above)
Comment (tsauerwein) [inline requirements.txt]: `2.3.0` is the latest version: https://pypi.python.org/pypi/elasticsearch

Thanks for the pointer.

File: c2corg_api/search/search_filters.py (hunk 1 above)
Comment (tsauerwein) [inline c2corg_api/search/search_filters.py]: That makes it easier, yes.

**Comment by tsauerwein** [comment]
Thanks for the reviews!

**Pull request merged**